message/validation/validation.go

```diff
@@ -96,6 +97,12 @@ func (mv *messageValidator) Validate(_ context.Context, peerID peer.ID, pmsg *pu
 
 	decodedMessage, err := mv.handlePubsubMessage(pmsg, time.Now())
 	if err != nil {
+		if mv.netCfg.Beacon.EstimatedCurrentSlot() == mv.netCfg.Beacon.FirstSlotAtEpoch(mv.netCfg.AlanForkEpoch) {
+			fullErr, ok := err.(Error)
+			if ok && errors.Is(fullErr.innerErr, ErrMalformedPubSubMessage) {
```

Comment:
```suggestion
			var fullErr Error
			if errors.As(err, &valErr) && fullErr.text == ErrMalformedPubSubMessage.text {
```

Reply:
you meant `fullErr` and not `valErr` right?

Reply:
Yes, sorry